src/components/HomePage/Forms/MovieForm.tsx

```diff
@@ -1,32 +1,77 @@
-import React, { FC } from 'react';
+import React, { FC, FormEvent } from 'react';
+import { useDispatch } from 'react-redux';
 import { Form } from 'reactstrap';
-import { MovieInfo } from '../../../types';
+import { addMovieCreator, updateMovieCreator } from '../../../redux/actions/actionCreators/actionCreators';
+import { MovieInfoDetails } from '../../../types';
 import { ResetButton, SubmitButton } from '../../shared';
 import './Form.scss';
 import { InputWithLabel } from './InputWithLabel';
 import { SelectInputGenre } from './SelectInputGenre';
 
-export const MovieForm: FC<{ movieInfo?: MovieInfo }> = ({ movieInfo }) => {
+export const MovieForm: FC<{ movieInfo?: MovieInfoDetails }> = ({ movieInfo }) => {
+  const dispatch = useDispatch();
+
+  const submit = (e: FormEvent<HTMLElement>) => {
+    e.preventDefault();
+    // todo check if mandatory fields filled
```

Comment:
Let's use https://formik.org/ for form validation, and etc.

Reply:
Ok, formik will be added in the next PR.

Reply:
https://github.com/Olga2157/react-movies-app/pull/7